### DISCUSSION TITLE
R1C1 as parametr

### DISCUSSION THREAD

**Discussion by Igniste** [JanMarvin/openxlsx2#831] [Ideas]
Hi, my friend had a case while combine multiple excel files (witch same structure and formulas) when some of them had R1C1 reference style parameter turn on, some have this parameter turn off. In case turn off read_xlsx or wb_load return "NA" values in columns with excel formulas. 
Would be great if there would be R1C1 reference style parameter in function like read_xlsx or wb_load, witch would turn on/off this parameter when read xlsx (same with write xlsx).

He resolve it with vba macro that turn on R1C1 reference style, if there is any existed R solution please let me know :)

**Comment by JanMarvin** [comment]
Hi @Igniste , mit sure I understand your request.

* So some workbooks use R1C1 notation and some others don’t? 
* And now „your friend“ (😉) wants to combine the sheets together? 
* Now the formulas for the R1C1 sheets are missing?

Afaik R1C1 is just an option for Excel to represent formulas and either excel uses R1C1 - which is the internal default - or A1 notation, but formulas in the xlsx xml files are always in A1 notation.
